testing/dv3d/TestManager.py

```diff
@@ -80,7 +80,7 @@ def __init__( self, name, **args ):
         self.image_name = os.path.join( self.test_dir, 'images', '.'.join( [ self.name, 'png' ] )  )
         filename = args.get( 'file', DefaultSampleFile )
         self.varnames = args.get( 'vars', [ DefaultSampleVar ] )
-        self.file_path = os.path.join( vcs.sample_data, filename )
+        self.file_path = os.path.join(*["/"]+vcs.__path__[0].split("/")[:-4]+["share","uvcdat","sample_data",filename] )
```

Comment:
@doutriaux1 why does the code in 83 not work on Mac? The new code seems too complicated.